### ISSUE TITLE
mypy could not recognize `sys` module's alias when checking version_info and  writing compact code

### ISSUE THREAD

**Issue by Hanaasagi** [python/mypy#6189]
#### Issue kind:
Bug Report.

#### Env:

- mypy 0.620
- Python 3.7.2

#### How to reproduce
Code as following:

```Python
# t.py
import sys as py_sys

if py_sys.version_info < (3, 0):
    unicode = unicode
    chr = unichr
else:
    unicode = str
```

exec `mypy t.py`, and use default config, it reports

```
t.py:5: error: Cannot determine type of 'unicode'
t.py:6: error: Name 'unichr' is not defined
```

But following code is ok.

```Python
# t.py
import sys

if sys.version_info < (3, 0):
    unicode = unicode
    chr = unichr
else:
    unicode = str
```

Master branch(mypy-0.660+dev.d1c8f27f696012085ca904e96f8b93b92b74919e mypy-extensions-0.4.1 typed-ast-1.1.1) has the same problem.

#### Except:

`mypy` can recognize `sys` module alias, and handle  compact code.

#### Actual:

If use a `sys` module's alias to get Python version, and write some compact code, it will check all code.

**Comment by ilevkivskyi** [comment]
PEP 484 clearly specifies what is expected to be understood statically https://www.python.org/dev/peps/pep-0484/#version-and-platform-checking, and what you do is not in the list of "simple version and platform checks", this is therefore low priority (if worth fixing at all).

**Comment by JukkaL** [comment]
This may be pretty easy to fix though.

**Comment by JukkaL** [comment]
> I don't want to introduce a separate mechanism for detecting early module aliases.

Yes, this would duplicate some logic. We could perhaps get away with a subset of the full alias handling logic, but we'd need to draw the line somewhere. Only supporting, say, `import sys as foo` wouldn't be that difficult, but then some users might want `from sys import version_info as _version_info` to work, etc. A general implementation would _not_ be simple.

**Comment by gvanrossum** [comment]
The typical pattern I've seen is that in one module, constants are defined, e.g. `PY3`, `WIN`, derived from `sys.version_info` or `sys.platform`. These are then imported into other modules and used for tests there, e.g. `if PY3: ...`

Unless we want to implement that, I don't think we should do anything here.
